tests/filecheck/mlir-conversion/with-mlir/dialects/bufferization/ops.mlir

```diff
@@ -3,11 +3,13 @@
 module{
     %t = "bufferization.alloc_tensor"() <{"operandSegmentSizes" = array<i32: 0, 0, 0>}> : () -> tensor<10x20x30xf64>
     %m = "test.op"() : () -> memref<30x20x10xf32>
-    %m_t = "bufferization.to_tensor"(%m) : (memref<30x20x10xf32>) -> tensor<30x20x10xf32>
+    %m_t = "bufferization.to_tensor"(%m) <{"restrict"}> : (memref<30x20x10xf32>) -> tensor<30x20x10xf32>
```

Comment:
might as well write this in custom syntax also IMO

Reply:
Done as suggested